shared/actions/people.tsx

```diff
@@ -54,12 +54,26 @@ const getPeopleData = (state, action: PeopleGen.GetPeopleDataPayload) => {
             verifyAllEmail: 'user@example.com',
             verifyAllPhoneNumber: '+1555000111',
           } as any)
+          let metadata
+          if (
+            avdlType === RPCTypes.HomeScreenTodoType.verifyAllEmail ||
+            avdlType === RPCTypes.HomeScreenTodoType.legacyEmailVisibility
+          ) {
+            metadata = Constants.makeTodoMetaEmail({
+              email: 'user@example.com',
+            })
+          } else if (avdlType === RPCTypes.HomeScreenTodoType.verifyAllPhoneNumber) {
+            metadata = Constants.makeTodoMetaPhone({
+              phone: '+1555000111',
```

Comment:
To test, either use a dev server, or set `debugTodo = true` and these values to unverified emails/phones on your account.